glide.yaml

```diff
@@ -66,4 +60,8 @@ import:
 - package: github.com/denisbrodbeck/machineid
   version: v1.0.1
 - package: github.com/google/uuid
-  version: v1.1.2
\ No newline at end of file
+  version: v1.1.2
+- package: github.com/adshao/go-binance
+  version: ^2.3.0
```

Comment:
let's use the exact version `2.3.0` instead of `^2.3.0` for better predictability. we use that pattern for other dependencies as well.